docs/src/contribute/core-rules.md

```diff
@@ -108,3 +108,13 @@ The rule naming conventions for ESLint are as follows:
 * Use dashes between words.
 * If your rule only disallows something, prefix it with `no-` such as `no-eval` for disallowing `eval()` and `no-debugger` for disallowing `debugger`.
 * If your rule is enforcing the inclusion of something, use a short name without a special prefix.
+
+## Frozen Rules
+
+When rules are feature complete, they are marked as frozen (indicated with ❄️ in the documentation). Rules are considered feature complete when the intended purpose of the rule has been fully implemented such that it catches 80% or more of expected violations and covers the majority of common exceptions. After that point, we expect users to use [disable comments](../use/configure/rules#using-configuration-comments-1) when they find an edge case that isn't covered.
```

Comment:
how about if we also suggest the users to change the rule locally according to their needs when they find an edge case  because that's what we suggest in issues mostly when they propose a change request?

Reply:
Good idea.